conf/fedora2-native.xml

```diff
@@ -66,13 +70,9 @@
     <bean id="namespacePrefixMapper" class="org.fcrepo.migration.foxml.NamespacePrefixMapper">
         <constructor-arg name="namespaceFile" type="java.io.File" ref="namespaceFile"/>
     </bean>
-
-    <bean id="fedora4Client" class="org.fcrepo.migration.f4clients.DefaultFedora4Client">
-        <constructor-arg name="repo" ref="repo" />
-    </bean>
-
-    <bean id="repo" class="org.fcrepo.client.impl.FedoraRepositoryImpl">
-        <constructor-arg name="repositoryURL" ref="fedora4Url" />
+    
+    <bean id="fedora4Client" class="org.fcrepo.migration.f4clients.StatelessFedora4Client">
+        <constructor-arg name="fcrepoBaseURL" ref="fedora4Url" />
```

Comment:
This isn't a huge thing, but it's nice to use the [constructor-arg namespace](http://docs.spring.io/spring/docs/3.1.0.M1/spring-framework-reference/html/beans.html#beans-c-namespace) instead: makes things a lot more readable.